CMakeLists.txt

```diff
@@ -46,6 +77,7 @@ if(WIN32 AND MSVC_VERSION LESS 1900)
   message(FATAL_ERROR "Building with Microsoft compiler needs Latest Visual Studio 2015 (Community or better)")
 endif()
 
+option(ENABLE_MASON "Use mason for dependencies" OFF)
```

Comment:
Added here (https://github.com/Project-OSRM/osrm-backend/commit/87d09f78e6584037b6e879705e6b6a65ecabd3a8#diff-af3b638bc2a3e6c650974192a53c7291R220) after LTO and GOLD checks have run. Let me know if that looks good.